tileserver/__init__.py

```diff
@@ -186,8 +189,14 @@ def __init__(self, layer_config, extensions, data_fetcher,
         self.redis_cache_index = redis_cache_index
         self.sqs_queue = sqs_queue
         self.buffer_cfg = buffer_cfg
+        self.formats = formats
         self.health_checker = health_checker
         self.add_cors_headers = add_cors_headers
+        self.metatile_size = metatile_size
+        if self.metatile_size:
```

Comment:
If someone is silly enough to set `metatile_size` to zero, the assertion below won't get tripped. Maybe check for `None` instead of truthiness?